src/usecases.test.js

```diff
@@ -0,0 +1,36 @@
+const usecases = require('./usecases')
+const assert = require('assert')
+
+describe('usecases', () => {
+    it('should create an empty list', () => {
+        assert.deepEqual(usecases.all, new Map())
+    })
+
+    it('should throw an error when no id is provided and the usecase has no description', () => {
+        assert.throws(() => usecases.add(class { }), /^Error: Herbarium requires an id for this usecase or an usecase with non empty description.$/)
+    })
+
+    it('should create an item with the provided id and usecase', () => {
+        const usecase = {
+            authorize() { }
+        }
+
+        const id = 'an id'
+
+        const addedUsecase = usecases.add(usecase, id)
+
+        assert.equal(addedUsecase.id, id)
+        assert.equal(addedUsecase.usecase, usecase)
+    })
+
+    it('should create an item with the usecase description as id when no id is provided', () => {
+        const usecase = {
```

Comment:
would be possible to use the actual herbs usecase implementation?

Reply:
Yes, but there would be necessary to add the dev-dependency of `@herbsjs/buchu`.

Reply:
I think it would be ok for `devDependencies`